google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/RetryInfoTest.java

```diff
@@ -368,6 +545,45 @@ private void verifyRetryInfoCanBeDisabled(Runnable runnable) {
     assertThat(attemptCounter.get()).isEqualTo(1);
   }
 
+  // Test the case where server does not return retry info
+  private void verifyNoRetryInfo(Runnable runnable, boolean operationRetryable) {
+    enqueueRetryableExceptionNoRetryInfo();
+
+    if (!operationRetryable) {
+      try {
+        runnable.run();
+        Assert.fail("non retryable operation should fail");
+      } catch (ApiException e) {
+        assertThat(attemptCounter.get()).isEqualTo(1);
+      }
```

Comment:
you should use Assert.assertThrows, it will print a better error message and make it easier to read the code

```assertThrows(ApiException.class, runnable)``